### ISSUE TITLE
Accessing Binary Files in plugin

### ISSUE THREAD

**Issue by ashutoshmadhwani** [Kong/kong-js-pdk#395]
I am writing a custom js plugin for kong where I need to access the binary files that are coming with request as multipart form data
Currently I am using the getRawBody function provided by kong-pdk, but I am getting following error

`2024/02/19 11:41:36 [warn] 1293#0: *2580 a client request body is buffered to a temporary file /var/run/kong/client_body_temp/0000000002"
2024/02/19 11:41:36 [error] 1293#0: *2580 [kong] init.lua:405 [clacks] /usr/local/share/lua/5.1/MessagePack.lua:126: attempt to get length of local 'str' (a nil value)`

Is there a way to resolve it
Basically I want to access the number of files uploaded, size of files and the file extension

**Comment by brentos** [comment]
Kong has a KB article about this, you need to increase the `client_body_buffer_size`:
https://support.konghq.com/support/s/article/Kong-plugin-produces-a-warning-a-client-request-body-is-buffered-to-a-temporary-file

**Comment by ashutoshmadhwani** [comment]
@brentos The request is directly coming to kong, there is no nginx cinfigured, Is there a way to configure this value in kong directly
